lib/compat/wordpress-6.6/post.php

```diff
@@ -11,9 +11,28 @@
 function gutenberg_add_excerpt_support_to_wp_block() {
 	add_post_type_support( 'wp_block', 'excerpt' );
 }
-
 add_action( 'init', 'gutenberg_add_excerpt_support_to_wp_block' );
 
+/**
+ * Add the rendering_mode property to the WP_Post_Type object.
+ * This property can be overwritten by using the post_type_default_rendering_mode filter.
+ *
+ * @param array  $args      Array of post type arguments.
+ * @return array Updated array of post type arguments.
+ */
+function gutenberg_post_type_default_rendering_mode( $args ) {
+	if (
+		( isset( $args['show_in_rest'] ) && $args['show_in_rest'] ) &&
+		( isset( $args['supports'] ) && in_array( 'editor', $args['supports'], true ) ) &&
+		( ! isset( $args['rendering_mode'] ) )
+	) {
+		$args['rendering_mode'] = 'post-only';
```

Comment:
```suggestion
		$args['default_rendering_mode'] = 'post-only';
```

I would suggest renaming the new `rendering_mode` property to `default_rendering_mode`. The reason why is that users are still able to change the rendering mode in the UI. But the word `rendering_mode` implies that it is static.

Reply:
Thanks, @fabiankaegy ! I've updated this to read `default_rendering_mode`